Sources/Purchasing/ProductsManager.swift

```diff
@@ -162,6 +162,7 @@ private extension ProductsManager {
                                                     errorMessage: errorMessage,
                                                     errorCode: errorCode,
                                                     storeKitErrorDescription: storeKitErrorDescription,
+                                                    storefront: self.systemInfo.storefront?.countryCode,
```

Comment:
This uses SK1 to calculate the storefront... I believe it should be ok? Lmk if there are any concerns though!

Reply:
Yeah, I think it's fine. We could use SK2's `storefront` but it's an `async` API, so it would require deeper changes. It can be done in a future PR I think